apps/LAMMPS/build_lammps_19Jan2022_ARCHER2_gcc_1020.md

```diff
@@ -0,0 +1,45 @@
+Building LAMMPS on ARCHER2 (GCC 10.2.0)
+===================================================
+
+These instructions are for building LAMMPS on ARCHER2 using the GNU compilers
+
+Download LAMMPS
+---------------
+
+Clone the latest stable version of LAMMPS from the GitHub repository:
+
+   `git clone -b stable https://github.com/lammps/lammps.git mylammps`
+
+Setup your environment
+----------------------
+
+Load the correct modules:
+
+   ```bash
+   module restore
+   module swap PrgEnv-cray PrgEnv-gnu 
+   module load cray-fftw module load cmake
```

Comment:
Typo on this line, should be either `module load cray-fftw cmake` or two lines

Reply:
Good spot -- fixed now.